mkdocs.yml

```diff
@@ -682,9 +688,7 @@ nav:
             - ProductStockRangeAggregation: search/aggregation_reference/productstockrange_aggregation.md
             - ProductPriceRangeAggregation: search/aggregation_reference/productpricerange_aggregation.md
             - ProductTypeTermAggregation: search/aggregation_reference/producttypeterm_aggregation.md
-            - SelectionTermAggregation: search/aggregation_reference/selectionterm_aggregation.md
             - TaxonomyEntryIdAggregation: search/aggregation_reference/taxonomyentryid_aggregation.md
```

Comment:
`TaxonomyEntryIdAggregation` appears two times. One in [Content aggregations](https://ez-systems-developer-documentation--2262.com.readthedocs.build/en/2262/search/aggregation_reference/aggregation_reference/#content-aggregations), one in [Product aggregations](https://ez-systems-developer-documentation--2262.com.readthedocs.build/en/2262/search/aggregation_reference/aggregation_reference/#product-aggregations).
As the left TOC is supposed to reflect those tables from main page, I decided to also keep it twice in this TOC.